content/intro-to-storybook/vue/pt/conclusion.md

```diff
@@ -3,31 +3,32 @@ title: 'Conclusão'
+Parabéns! Você criou sua primeira interface do usuário no Storybook. Ao longo do caminho, você aprendeu a criar, compor, testar e implantar componentes da interface do usuário. Se você seguiu o tutorial, seu repositório e o Storybook implantado devem ficar assim:
 
 [📕 **GitHub repo: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
 [🌎 **Storybook implementado**](https://clever-banach-415c03.netlify.com/)
 
-O Storybook é uma ferramenta bastante poderosa para React, Vue, Angular.
-É composta por uma comunidade de desenvolvimento próspera e um número grande de extras. Esta introdução somente arranha a superfície das potencialidades existentes. Estamos confiantes que depois da adoção do Storybook, irá ficar impressionado com o quanto é produtivo a construção de interfaces de utilizador duradouros.
+O Storybook é uma ferramenta poderosa para React, Vue e Angular.
+Possui uma comunidade de desenvolvedores próspera e uma grande variedade de complementos. Esta introdução arranha a superfície do que é possível. Estamos confiantes de que, ao adotar o Storybook, você ficará impressionado com a produtividade da criação de UIs duráveis.
 
 ## Saber mais
 
-Pretende saber mais? Aqui ficam alguns recursos que irão ajudar.
+Pretende saber mais? Aqui ficam algumas dicas que irão ajudar.
 
-- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contem a documentação da API, links da comunidade e a galeria de extras.
+- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contem a documentação da API, links da comunidade e a galeria de complementos.
 
 - [**Manual de testes visuais**](https://blog.hichroma.com/the-delightful-storybook-workflow-b322b76fd07)
-  enumera as boas práticas de fluxo de trabalho, usadas por equipas que operam a alta velocidade, tal como por exemplo a SquareSpace, a Major League Soccer, a Discovery Network e Apollo GraphQL.
+  enumera as boas práticas de fluxo de trabalho, usadas por equipes que operam a alta velocidade, tal como por exemplo a SquareSpace, a Major League Soccer, a Discovery Network e Apollo GraphQL.
 
 - [**Visual Testing Handbook**](https://www.chromaticqa.com/book/visual-testing-handbook) aprofunda o uso de Storybook para testes visuais. Livro eletrónico livre de 31 páginas.
 
 ## Quem fez LearnStorybook.com?
 
-O texto, código e produção foram contribuições feitas pela [Chroma](http://blog.hichroma.com/). O tutorial foi inspirado na popular [Serie de tutoriais GraphQL + React](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858)
 
-Se pretender mais tutoriais e artigos tal como este? Subscreva a mailing list da Chroma.
+O texto, código e produção foram contribuídos por [Chroma](http://blog.hichroma.com/). O tutorial foi inspirado no popular Chroma [GraphQL + React tutorial series](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858).
```

Comment:
A better wording for this could be something like:
"O texto, código e produção foram contribuídos pela [Chroma](http://blog.hichroma.com/). O tutorial foi inspirado pela [série de tutoriais populares](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858). Quer mais tutoriais e artigos como este? Subscreva a mailing list da Chroma."

Reasoning behind this:
- Usage of "pela" instead of "por" is accurate for both readers, does not break the translation and technically yelds the same result.
- moved in "série" inside the link so that does not break the wording altogether. 
- Usage of "Subscreva"is not unreasonable and it's valid for both readers. I think that by changing it we're not bringing anything new to the table.